### PR TITLE
Rename 'template' to 'sample' in filenames, codebase, & documentation

### PR DESCRIPTION
**Pull request by kelsk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,42 +15,42 @@ The `.cctemplate` is a [Template Descriptor](https://cloud.google.com/code/docs/
...
 
-A sample Node.js template with a frontend and backend service. This template uses MongoDB as a database and runs on Kubernetes. You can see an entry for this template in the `.cctemplate` file.
+A Node.js sample with a frontend and backend service. This sample uses MongoDB as a database and runs on Kubernetes. You can see an entry for this sample in the `.cctemplate` file.
 
 ---
 
 ## Usage
 
-To see an example of how the Custom Templates feature of Cloud Code works, you can configure your IDE to recognize this repository and create applications based on the templates within. Follow the steps below to see Custom Templates in action!
+To see an example of how the Custom Sample feature of Cloud Code works, you can configure your IDE to recognize this repository and create applications based on the sample within. Follow the steps below to see Custom Sample in action!
```
Comment (abbycar) [inline README.md:41]: "Follow the steps below to see Custom Sample**s**?

Comment (abbycar) [reply README.md:41]: or "this feature in action"

File: README.md [hunk 2]
```diff
@@ -15,42 +15,42 @@ The `.cctemplate` is a [Template Descriptor](https://cloud.google.com/code/docs/
...
+2. Import the Custom Sample repo on [IntelliJ](https://cloud.google.com/code/docs/intellij/create-app-from-custom-template?utm_source=ext&utm_medium=partner&utm_campaign=CDR_kri_gcp_cloudcodereadmes_012521&utm_content=-) or [VS Code](https://cloud.google.com/code/docs/vscode/create-app-from-custom-template?utm_source=ext&utm_medium=partner&utm_campaign=CDR_kri_gcp_cloudcodereadmes_012521&utm_content=-), using this repo's Git URL: `https://github.com/GoogleCloudPlatform/cloud-code-custom-samples-example.git`
 
-You can create your own Custom Template repo using this repository as a base.
+You can create your own Custom Sample repo using this repository as a base.
 
 1. Clone this repository
 
-2. Create a new directory containing your own template (or make changes to any of the sample templates)
+2. Create a new directory containing your own sample (or make changes to any of the sample samples)
```
Comment (abbycar) [inline README.md:52]: Can we reword "sample samples?" :P

File: kubernetes-custom-sample/nodejs/README.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
-# Kubernetes Custom Template
+# Kubernetes Custom Sample
 
-This Custom Template creates a basic Kubernetes web application with a frontend and backend service. The template files include logic to make HTTP requests, read from and write to a database, and display content. The template can be customized to build a dynamic application.
+This Custom Sample creates a basic Kubernetes web application with a frontend and backend service. The sample files include logic to make HTTP requests, read from and write to a database, and display content. The sample can be customized to build a dynamic application.
```
Comment (abbycar) [inline kubernetes-custom-sample/nodejs/README.md:3]: lower case

File: kubernetes-custom-sample/nodejs/README.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 - [Deploying with IntelliJ](#deploying-with-intellij)
 - [Sign up for user research](#sign-up-for-user-research)
 
 ---
 
-## Template contents
+## Sample contents
 
-The Kubernetes Custom Template contains a **skaffold.yaml** and Kubernetes manifest yamls for each service. When you deploy the app with Cloud Code, it uses [skaffold](https://skaffold.dev/docs/) to build an image and deploy the project's Kubernetes manifests.
+The Kubernetes Custom Sample contains a **skaffold.yaml** and Kubernetes manifest yamls for each service. When you deploy the app with Cloud Code, it uses [skaffold](https://skaffold.dev/docs/) to build an image and deploy the project's Kubernetes manifests.
```
Comment (abbycar) [inline kubernetes-custom-sample/nodejs/README.md:21]: lowercase

File: kubernetes-custom-sample/nodejs/README.md [hunk 5]
```diff
@@ -49,17 +49,17 @@ The backend service manages requests to a MongoDB database.
 
 ## Examples
 
-For an example of an application built using this template, see the [Cloud Code "Guestbook" sample](https://github.com/GoogleCloudPlatform/cloud-code-samples/tree/master/nodejs/nodejs-guestbook).
+For an example of an application built using this sample, see the [Cloud Code "Guestbook" sample](https://github.com/GoogleCloudPlatform/cloud-code-samples/tree/master/nodejs/nodejs-guestbook).
 
 ![Guestbook display](./img/my-guestbook.png)
 
-Are you using the Kubernetes Custom Template to build an app? Let us know by joining the **#cloud-code** channel in the [Google Cloud Platform Slack community](https://join.slack.com/t/googlecloud-community/shared_invite/zt-erdf4ity-8ZMUQ18DYV~5hkbZ~gCswg)!
+Are you using the Kubernetes Custom Sample to build an app? Let us know by joining the **#cloud-code** channel in the [Google Cloud Platform Slack community](https://join.slack.com/t/googlecloud-community/shared_invite/zt-erdf4ity-8ZMUQ18DYV~5hkbZ~gCswg)!
```
Comment (abbycar) [inline kubernetes-custom-sample/nodejs/README.md:56]: lowecase

File: README.md (hunk 2 above)
Comment (kschaab) [reply README.md:52]: Agreed, this works:
```suggestion
2. Create a new directory containing your own sample (or make changes to any of the samples)
```

**Review by abbycar — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (kelsk) [reply README.md:52]: Whoops, this was a casualty of being too careless with "find and replace". Thanks for the catch!

File: README.md (hunk 1 above)
Comment (kelsk) [reply README.md:41]: "this feature in action" is perfect 👍

**Review by abbycar — APPROVED** [review]

**Pull request merged**